server/src/services/firewood-permit-service.es6

```diff
@@ -0,0 +1,167 @@
+const htmlToText = require('html-to-text');
+const moment = require('moment-timezone');
+const uuid = require('uuid/v4');
+const zpad = require('zpad');
+
+const email = require('../email/email-util.es6');
+const forestDb = require('../models/forests.es6');
+const vcapConstants = require('../vcap-constants.es6');
+const paygov = require('./paygov');
+const permitSvgService = require('./christmas-trees-permit-svg-util.es6');
+
+const firewoodPermitService = {};
+
+/**
+ * @function translatePermitFromClientToDatabase - Private function to translate request data to the database json object
+ * @param {Object} input - christmas trees permit object from database
+ * @return {Object} - formatted permit object
+ */
+firewoodPermitService.translatePermitFromClientToDatabase = (permit, forest) => ({
+  permitId: uuid(),
+  firstName: permit.firstName,
+  lastName: permit.lastName,
+  emailAddress: permit.emailAddress,
+  quantity: permit.quantity,
+  // TODO update to reflect real cost
```

Comment:
Can we address this `todo` now?

Reply:
Out of scope for the issue per melissa and eric